src/core/model/model.ts

```diff
@@ -190,7 +190,7 @@ export class Model extends Config {
   }
 
   /**
-   * Check if the model has parameter.
+   * Check if the model has parameters.
```

Comment:
It actually checks if the request parameter id is provided.

Reply:
Ok, I adressed this in the commits, too.